test/integration/scene-stream-access/reset-scene-stream-access-handler.spec.ts

```diff
@@ -0,0 +1,276 @@
+import { IngressInfo } from 'livekit-server-sdk/dist/proto/livekit_ingress'
+
+test('PUT /scene-stream-access - resets streaming access for scenes', ({ components, stubComponents }) => {
+  const FOUR_DAYS = 4 * 24 * 60 * 60
+  const placeId = `place-id-stream-access`
+  const placeWorldId = `place-id-world-stream-access`
+  let cleanup: TestCleanup
+
+  type Metadata = {
+    identity: string
+    realm: {
+      serverName: string
+      hostname: string
+      protocol: string
+    }
+    sceneId: string
+    parcel: string
+  }
+
+  let metadataLand: Metadata
+  let metadataWorld: Metadata
+  let mockIngress: IngressInfo
+  let mockSceneStreamAccess: any
+
+  beforeAll(async () => {
+    cleanup = new TestCleanup(components.database)
+  })
+
+  beforeEach(async () => {
+    mockIngress = {
+      id: 'mock-ingress-id',
+      name: 'mock-ingress',
+      url: 'rtmp://mock-stream-url',
+      streamKey: 'mock-stream-key',
+      ingressId: 'mock-ingress-id'
+    } as IngressInfo
+
+    mockSceneStreamAccess = {
+      id: 'mock-access-id',
+      place_id: placeId,
+      streaming_url: 'rtmp://mock-stream-url',
+      streaming_key: 'mock-stream-key',
+      ingress_id: 'mock-ingress-id',
+      created_at: Date.now(),
+      active: true
+    }
+
+    metadataLand = {
+      identity: owner.authChain[0].payload,
+      realm: {
+        serverName: 'test-realm',
+        hostname: 'https://peer.decentraland.zone',
+        protocol: 'https'
+      },
+      parcel: '10,20',
+      sceneId: 'test-scene'
+    }
+
+    metadataWorld = {
+      identity: owner.authChain[0].payload,
+      realm: {
+        serverName: 'name.dcl.eth',
+        hostname: 'https://worlds-content-server.decentraland.org/',
+        protocol: 'https'
+      },
+      parcel: '20,20',
+      sceneId: 'test-scene'
+    }
+
+    jest.spyOn(handlersUtils, 'validate').mockResolvedValue({
+      identity: owner.authChain[0].payload,
+      realm: {
+        serverName: 'test-realm',
+        hostname: 'https://peer.decentraland.zone',
+        protocol: 'https'
+      },
+      parcel: '10,20',
+      sceneId: 'test-scene'
+    })
+
+    stubComponents.places.getPlaceByParcel.resolves({
+      id: placeId,
+      positions: ['10,20'],
+      owner: owner.authChain[0].payload
+    } as PlaceAttributes)
+
+    stubComponents.places.getPlaceByWorldName.resolves({
+      id: placeWorldId,
+      world_name: 'name.dcl.eth',
+      owner: owner.authChain[0].payload
+    } as PlaceAttributes)
+
+    stubComponents.lands.getLandUpdatePermission.resolves({ owner: true, operator: false })
+    stubComponents.sceneManager.isSceneOwnerOrAdmin.resolves(true)
+    stubComponents.livekit.getSceneRoomName.resolves(`test-realm:test-scene`)
+    stubComponents.livekit.getWorldRoomName.resolves(`name.dcl.eth`)
+  })
+
+  afterEach(async () => {
+    await cleanup.cleanup()
+    jest.restoreAllMocks()
+  })
+
+  it('returns 200 with new streaming access when user is owner', async () => {
```

Comment:
When the user is land owner, right?
```suggestion
  it('returns 200 with new streaming access when user is land owner', async () => {
```